Apps/Examples/Examples/All Examples/SnapshotterExample.swift

```diff
@@ -8,65 +8,79 @@ public class SnapshotterExample: UIViewController, ExampleProtocol {
     internal var mapView: MapView!
     public var snapshotter: Snapshotter!
     public var snapshotView: UIImageView!
+    private var snapshotting = false
 
-    override public func viewDidLoad() {
-        super.viewDidLoad()
+    override public func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
 
         // Create a vertical stack view to hold both the map view and the snapshot.
         let stackView = UIStackView(frame: view.safeAreaLayoutGuide.layoutFrame)
         stackView.axis = .vertical
-        stackView.distribution = .fillEqually
+        stackView.distribution = .fill
+        stackView.alignment = .center
         stackView.spacing = 12.0
 
-        let testRect = CGRect(x: 0, y: 0, width: view.bounds.width, height: view.bounds.height / 2)
+        let testRect = CGRect(x: 0, y: 0, width: view.bounds.width/2, height: view.bounds.height / 2)
```

Comment:
what is `testRect`? Given this is a public example, we should use a stronger more descriptive variable name here